openslides/users/signals.py

```diff
@@ -113,79 +112,88 @@ def create_builtin_groups_and_admin(**kwargs):
         return
 
-    # Anonymous (pk 1) and Registered (pk 2)
-    ct_core = ContentType.objects.get(app_label='core', model='customslide')
-    perm_11 = Permission.objects.get(content_type=ct_core, codename='can_see_projector')
-    perm_12 = Permission.objects.get(content_type=ct_core, codename='can_see_dashboard')
-
-    ct_agenda = ContentType.objects.get(app_label='agenda', model='item')
-    ct_speaker = ContentType.objects.get(app_label='agenda', model='speaker')
-    perm_13 = Permission.objects.get(content_type=ct_agenda, codename='can_see_agenda')
-    perm_14 = Permission.objects.get(content_type=ct_agenda, codename='can_see_orga_items')
-    can_speak = Permission.objects.get(content_type=ct_speaker, codename='can_be_speaker')
-
-    ct_motion = ContentType.objects.get(app_label='motion', model='motion')
-    perm_15 = Permission.objects.get(content_type=ct_motion, codename='can_see_motion')
-
-    ct_assignment = ContentType.objects.get(app_label='assignment', model='assignment')
-    perm_16 = Permission.objects.get(content_type=ct_assignment, codename='can_see_assignments')
-
-    ct_users = ContentType.objects.get(app_label='users', model='user')
-    perm_users_can_see_name = Permission.objects.get(content_type=ct_users, codename='can_see_name')
-    perm_users_can_see_extra_data = Permission.objects.get(content_type=ct_users, codename='can_see_extra_data')
-
-    ct_mediafile = ContentType.objects.get(app_label='mediafile', model='mediafile')
-    perm_18 = Permission.objects.get(content_type=ct_mediafile, codename='can_see')
-
-    base_permission_list = (
-        perm_11,
-        perm_12,
-        perm_13,
-        perm_14,
-        perm_15,
-        perm_16,
-        perm_users_can_see_name,
-        perm_users_can_see_extra_data,
-        perm_18)
+    permission_dict = {}
+    permission_strings = (
+        'agenda.can_be_speaker',
+        'agenda.can_manage_agenda',
+        'agenda.can_see_agenda',
+        'agenda.can_see_orga_items',
+        'assignment.can_manage_assignments',
+        'assignment.can_nominate_other',
+        'assignment.can_nominate_self',
+        'assignment.can_see_assignments',
+        'config.can_manage',
+        'core.can_manage_projector',
+        'core.can_manage_tags',
+        'core.can_see_dashboard',
+        'core.can_see_projector',
+        'core.can_use_chat',
+        'mediafile.can_manage',
+        'mediafile.can_see',
+        'mediafile.can_upload',
+        'motion.can_create_motion',
+        'motion.can_manage_motion',
+        'motion.can_see_motion',
+        'motion.can_support_motion',
+        'users.can_manage',
+        'users.can_see_extra_data',
+        'users.can_see_name', )
+    for permission_string in permission_strings:
+        app_label, codename = permission_string.split('.')
+        permission_dict[permission_string] = Permission.objects.get(
+            content_type__app_label=app_label, codename=codename)
```

Comment:
Proposal: find a way only to use one query instead of one query per permission.

This is not so important, because this function is not called so often.